.env

```diff
@@ -0,0 +1,7 @@
+SERVER_PORT=3001
+DB_DATABASE_NAME=clubs
+DB_USERNAME=loolabs
+DB_PASSWORD=loolabs
+DB_PORT=5432
+DATABASE_URL=postgresql://loolabs:loolabs@db/clubs
+IS_DATABASE_LOCAL=true
```

Comment:
Why exactly do we need to commit this file? From what goes on in `integration-test.yml`, it looks like these env variables aren't used in the integration test

Reply:
This is just committing the file for local dev purposes from our earlier discussions.

Reply:
we'll regret adding `.env` the moment we need to put a 3rd party API key into this file

Reply:
Removed